content/strategy-goals/strategy/code-exploration/index.md

```diff
@@ -0,0 +1,29 @@
+# Code Exploration
+
+This page contains the vision for Code Exploration in Sourcegraph (and a little beyond).
+
+Note this is not a roadmap – we won’t realize all of these in the next year. Not all of these things will pass the prioritization and effort/value test when it comes to roadmap planning.
+It's our north star, a shared vision of what we’re building towards, with every improvement we’ll build over the next year taking us closer towards it.
+One that will put and keep our product offering in a unique, defendable position in the market, and won’t leave users choosing other products over us (whether it is because they’re better, or because they’re “good enough”).
```

Comment:
This all shows up as a single paragraph, not sure if you intended for there to be line breaks since there are some newlines in here!

Reply:
Yes, intended. Breaking paragraphs up into one sentence per line helps Git resolve merge conflicts and makes suggestions in GitHub easier.

Reply:
Sounds good! I didn't notice it anywhere else so just wanted to double check. :)